apps/samples/basic/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@workleap/typescript-configs/web-application.json",
+  "exclude": ["dist", "node_modules"],
+  "include": ["**/*", "../../../types"],
+  "compilerOptions": {
+        "paths": {
+            "@hopper-ui/components": ["../../../packages/components/src/index.ts"],
```

Comment:
[Question] Is this mandatory if it's already in the dependencies?

Reply:
When you add a dependency:workspace:*, it refers to the dist folder. We add this path so if we click on "go to implementation" of a function or symbol, it redirects to the source code